.travis.yml

```diff
@@ -1,9 +1,18 @@
 sudo: false
-cache: yarn
 language: node_js
 node_js:
   - "node"
   - "4.8"
+cache: yarn
 script:
    - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then yarn run test:unit:cov; fi'
    - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then yarn run test:cov; fi'
+deploy:
+  provider: npm
+  api_key:
+    secure: "fpsIqEZNvzcS2g91vwuR8UJo+4Ok6xGuVV/uL5nNia+8cnA+cSfQEmPm4w+0gB3F1UIOGs9ItuMqcBlZOMe7oZxr9Wi+J05LfZGL4cibRXU+ydpnQ0d5Si7/p4W3vskA+SnolDjHPCHAVxZiyRUXswuhMnajbrAejeBvS2n0WF4="
+  email: "systems@mollie.com"
```

Comment:
Check this with @martijnpolder. This email address is already publicly available through the NPM registry, so it's fine to use it in `.travis.yml`